code/modules/surgery/organs/organ_internal.dm

```diff
@@ -126,6 +126,51 @@
 /obj/item/organ/proc/OnEatFrom(eater, feeder)
 	useable = FALSE //You can't use it anymore after eating it you spaztic
 
+/* 
+ * On accidental consumption, cause organ damage and check if they like eating organs
+ */
+/obj/item/organ/on_accidental_consumption(mob/living/carbon/M, mob/living/carbon/user, obj/item/source_item, discover_after = TRUE)
+	if(organ_flags & ORGAN_SYNTHETIC) //Future idea: Make synthetic organs special
+		return ..()
+
+	if(organ_flags & ORGAN_FROZEN) //Future idea: Make frozen organs special
+		return TRUE
+
+	applyOrganDamage(25)
+	OnEatFrom(M, user)
+	if(istype(src, /obj/item/organ/brain)) //brain takes some extra damage
+		applyOrganDamage(25)
+		M.adjust_disgust(100) 
```

Comment:
what if I'm a species that likes raw meat, or a high-functioning zombie or something?

Reply:
also, hm, yes, chainstun vomiting

seriously, 100 disgust is a LOT. disgust decays at a rate of 0.5 disgust/2 seconds (or 0.25 disgust/second) for most people, and if your disgust level is 50 or more, you roll a [5+0.05*disgust]% chance of puking/getting hardstunned every 2 seconds. there's a reason why things like the badblink mutation have a bad reputation.

I guess what I'm trying to say is, reduce this number.

Reply:
meant to add a zombie check and forgot, also meant to lower the disgust but forgot, thanks (brain stuff was kinda the last thing I did)